PRS2001-3000/PRS2311Babnuda.xml

```diff
@@ -45,23 +45,24 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
          <change who="PL" when="2019-03-27">removed occupation containing St, Saint, bibl, biblical or prophet, added keyword. See https://github.com/BetaMasaheft/Documentation/issues/1104</change>
+         <change who="MKr" when="2020-11-03">Added name in Ethiopic and info</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
-               <persName>
-                  
-                  Babnūda
-               </persName>
-               <persName type="alt">Bäfnotyos</persName>
-               <persName type="alt">Bafnutyus</persName>
-               <persName type="alt">Pafnuti,  Paphnoutios</persName>
-               <!-- St.-->
-               <floruit> 15 Yäkkatit </floruit>
+            <person sex="1" sameAs="wd:Q12757084">
+                <persName xml:lang="eng" xml:id="n1">Paphnoutios</persName>
+                <persName xml:lang="gez" xml:id="n2">ባብኖዳ፡</persName>
+                <persName xml:lang="ar" corresp="#n2">Babnoda</persName>
```

Comment:
why is the lang ar? Every transliterated form should get type standardized (see also below)